RELEASE_NOTES.md

```diff
@@ -1,4 +1,6 @@
-#### 1.3.1.0 -
+#### 1.3.2 -
```

Comment:
You have 4 number to work with in a version. You could match the first two to the product version of `3.1` and `4.0`. The 3 number would be the number that you increment for stable releases.